docs/epayment.v1.yml

```
@@ -246,7 +211,8 @@ components:
           pattern: '^[0-9-]{10,15}$'
         sub:
           type: string
-          description: The user subscription identifier received in `$.profile.sub` or from the Vipps Login API
+          description: |
+            The user subscription identifier received in the `$.profile.sub` property from the response body or from the Vipps Login API
```

The `$` syntax is not needed for plain JSON, I think?
```suggestion
            The user subscription identifier received in the `profile.sub` property from the response body or from the Vipps Login API
```